js_modules/dagit/.yarn/plugins/@yarnpkg/plugin-engines.cjs

```diff
@@ -0,0 +1,9 @@
+return plugin;
+}
+};
```

Comment:
Added this plugin so that we can enforce the Node version in yarn via `package.json`.

I haven't enabled that restriction yet in this PR because I want to make such a change easy to revert if something unexpected comes up.